wye/workshops/forms.py

```diff
@@ -3,7 +3,7 @@
 
 from wye.base.widgets import CalendarWidget
 
-from .models import Workshop
+from . import models
```

Comment:
You can send a separate PR for that.

On Sun, Oct 25, 2015 at 10:29 PM, Taranjeet Singh <notifications@github.com>
wrote:

> In wye/workshops/forms.py
> <https://github.com/pythonindia/wye/pull/108#discussion_r42947126>:
>
> > @@ -3,7 +3,7 @@
> >
> >  from wye.base.widgets import CalendarWidget
> >
> > -from .models import Workshop
> > +from . import models
>
> @kracekumar <https://github.com/kracekumar> : I see many such imports in
> other views and models. I think I should create a new branch and fix all
> those first, so that everyone else avoids such things. What say?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pythonindia/wye/pull/108/files#r42947126>.
>



-- 
Regards
Kracekumar Ramaraj
http://kracekumar.com